### ISSUE TITLE
GameGuru MAX - Bug - Blank Area in Character Creator Causes Crash

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#3516]
See video:

https://user-images.githubusercontent.com/36735507/181826889-2f1d6f7c-fd46-476a-8b7f-9d38bcb1b52d.mp4

There are several of these blank areas in a few places (heads, hats, shoes, etc.).

**Comment by davetgc** [comment]
I'm not seeing this in the latest build, but have seen it before while converting older Classic characters to work in MAX. Are you doing anything like that?

**Comment by MonkeyFrogStudio** [comment]
@davetgc Nope. Not doing any character work, actually (nothing from Classic, either). My guess is that I am missing something. In fact, for some reason, the future soldiers (whatever they are called - the male and female with the sci-fi armor) are now missing from my installation. I think these blank areas are for parts pertaining to them? One reason I say this is they are not completely missing. I'll try to explain.

If I go to Add and Characters, the sci-fi characters are not there to add to my level. However, if I go to the Character Creator, PARTS of them are there, but not all parts. In other words, if I go to change aspects of one of the soldiers, I can access the boots that belong to the sci-fi soldiers, and the helmet, etc. So, perhaps the few blank areas that are causing this crash are related to the sci-fi soldiers "missing" (but also not missing - if that makes sense).

**Comment by davetgc** [comment]
Ok, that sounds like the problem. Thanks for the update.

**Comment by ZakJudges** [comment]
@MonkeyFrogStudio Thank you for reporting this issue. I have added a fix which should appear in an experimental build in the near future :)

**Comment by MonkeyFrogStudio** [comment]
Fixed! Thank you!
